dashboard_documentation.md

```diff
@@ -69,6 +69,78 @@ Create New Site
    <img class="my-3" src="/images/plant_site.png"/>
 
 
+Create New Aggregate
+--------------------
+{: .anchor}
+
```

Comment:
I think we need to say 1 or 2 sentences about how aggregates work. First define metadata, then associate *preexisting* observations with it and/or create *new* forecasts for it.